pkg/agent/openflow/client.go

```diff
@@ -40,7 +40,7 @@ type Client interface {
 	// be called to ensure that the set of OVS flows is correct. All flows programmed in the
 	// switch which match the current round number will be deleted before any new flow is
 	// installed.
-	Initialize(roundInfo types.RoundInfo, config *config.NodeConfig, networkconfig *config.NetworkConfig) (<-chan struct{}, error)
+	Initialize(roundInfo types.RoundInfo, config *config.NodeConfig, networkconfig *config.NetworkConfig, egressExceptConfig *config.EgressExceptConfig) (<-chan struct{}, error)
```

Comment:
It seems we only need to pass `exceptCIDRs` to `InstallExternalFlows`, which would make the code more readable as the signature shows we are telling `InstallExternalFlows` which CIDRs should not be regarded as external addresses.
It can also save adding another dependency to the OpenFlow `client` struct which is already more complex than it should be.

Reply:
Use `EgressConfig` would be more extensible, perhaps we will add protocol into the `EgressConfig` for https://github.com/antrea-io/antrea/issues/2671

Reply:
After some considering, I agreed `exceptCIDRs` is fine, we can add `protocol` into `exceptCIDRs` for #2671

Reply:
Agreed, I will fix this logic.